### PR TITLE
grep, ps, unzip: add Chinese translation

### PR DESCRIPTION
**Pull request by pan2013e**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by pan2013e** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/7424043992) for this PR failed with the following error(s):
> 
> ```
> pages.zh/common/ps.md:32:16 MD047/single-trailing-newline Files should end with a single newline character
> pages.zh/common/ps.md:1: TLDR009 Page should contain a newline at end of file
> ```
> 
> Please fix the error(s) and push again.

Fixed it :)

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.zh/common/grep.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- 在文件中查找模式：
+
+`grep "{{模式字符串}}" {{路径/到/文件}}`
+
+- 在文件中精确地查找字符串（禁用正则表达式）：
+
+`grep --fixed-strings "{{字符串}}" {{路径/到/文件}}`
+
+- 在指定目录下的所有文件中递归地查找模式，显示匹配的行号并忽略二进制文件:
```
Comment (blueskyson) [inline pages.zh/common/grep.md:14]:
```suggestion
- 在指定目录下的所有文件中递归地查找模式，显示匹配的行号并忽略二进制文件：
```

File: pages.zh/common/ps.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# ps
+
+> 提供正在运行的进程的信息。
+> 更多信息: <https://manned.org/ps>.
+
+- 列出所有正在运行的进程:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:6]:
```suggestion
- 列出所有正在运行的进程：
```

File: pages.zh/common/ps.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`ps aux`
+
+- 列出所有正在运行的进程，包括完整的命令字符串:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:10]:
```suggestion
- 列出所有正在运行的进程，包括完整的命令字符串：
```

File: pages.zh/common/ps.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`ps auxww`
+
+- 查找与字符串匹配的进程:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:14]:
```suggestion
- 查找与字符串匹配的进程：
```

File: pages.zh/common/ps.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`ps aux | grep {{字符串}}`
+
+- 以 extra full 格式列出当前用户的所有进程:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:18]:
```suggestion
- 以 extra full 格式列出当前用户的所有进程：
```

File: pages.zh/common/ps.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`ps --user $(id -u) -F`
+
+- 以树形方式列出当前用户的所有进程:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:22]:
```suggestion
- 以树形方式列出当前用户的所有进程：
```

File: pages.zh/common/ps.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`ps --user $(id -u) f`
+
+- 获取一个进程的父进程 ID:
+
+`ps -o ppid= -p {{进程 ID}}`
+
+- 按内存使用量对进程进行排序:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:30]:
```suggestion
- 按内存使用量对进程进行排序：
```

File: pages.zh/common/ps.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+- 以 extra full 格式列出当前用户的所有进程:
+
+`ps --user $(id -u) -F`
+
+- 以树形方式列出当前用户的所有进程:
+
+`ps --user $(id -u) f`
+
+- 获取一个进程的父进程 ID:
```
Comment (blueskyson) [inline pages.zh/common/ps.md:26]:
```suggestion
- 获取一个进程的父进程 ID：
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**